net/node/node.go

```diff
@@ -658,8 +659,10 @@ func (node *node) ConnectNeighbors() {
 		if !node.IsChordAddrInNeighbors(chordNbr.Id) {
 			chordNbrAddr, err := chordNbr.NodeAddr()
 			if err != nil {
+				log.Error("Get chord neighbor node addr error:", err)
```

Comment:
There was logged err in chordNbr.NodeAddr() already.  [NodeAddr()](https://github.com/nknorg/nkn/blob/master/net/chord/vnode.go#L26)
Why log it again at here?

Reply:
You are right, removing it